### PR TITLE
Update users-get.md

### PR DESCRIPTION
**Pull request by TopicElite**
### Prerequisites for Code Changes

* [x] This pull request follows the code style of the project
* [x] I have tested this feature

...

### Changes Proposed

* Add Kotlin Code to "print user list - search by id"

...

### CODE REVIEW

File: content/docs/rest-helix/users-get.md [hunk 1]
```diff
@@ -57,7 +57,10 @@ resultList.getUsers().forEach(user -> {
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-
+val resultList = twitchClient!!.helix.getUsers(null, listOf("149223493"), null).execute()
+resultList.users.forEach(Consumer { user: User? ->
+	println(user)
+})
```
Comment (iProdigy) [inline content/docs/rest-helix/users-get.md:63]:
```suggestion
resultList.users.forEach { user ->
	println(user)
}
```

**Review by iProdigy — COMMENTED** [review]
Thanks for your contribution!

File: content/docs/rest-helix/users-get.md [hunk 2]
```diff
@@ -57,7 +57,10 @@ resultList.getUsers().forEach(user -> {
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-
+val resultList = twitchClient!!.helix.getUsers(null, listOf("149223493"), null).execute()
```
Comment (PhilippHeuer) [inline content/docs/rest-helix/users-get.md:60]:
```suggestion
val resultList = twitchClient.helix.getUsers(null, listOf("149223493"), null).execute()
```
we can also remove the two `!!` as we don't have this in the other examples.

Comment (stachu540) [reply content/docs/rest-helix/users-get.md:60]: there is no need to be a null safety / null checks

File: content/docs/rest-helix/users-get.md (hunk 1 above)
Comment (stachu540) [reply content/docs/rest-helix/users-get.md:63]: Consumer have a [SAM interfaces](https://kotlinlang.org/docs/fun-interfaces.html) in code.

**Review by stachu540 — APPROVED** [review]
I have adding also a groovy examples. In any case... LGTM

**Review by iProdigy — APPROVED** [review]
LGTM! thanks @stachu540 for including the groovy example as well

**Review by PhilippHeuer — APPROVED** [review]

**Pull request merged**